flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/expressions/literals.scala

```diff
@@ -60,7 +60,6 @@ case class Literal(value: Any, resultType: TypeInformation[_]) extends LeafExpre
     case _@SqlTimeTypeInfo.TIMESTAMP => value.toString + ".toTimestamp"
     case _@TimeIntervalTypeInfo.INTERVAL_MILLIS => value.toString + ".millis"
     case _@TimeIntervalTypeInfo.INTERVAL_MONTHS => value.toString + ".months"
-    case _@RowIntervalTypeInfo.INTERVAL_ROWS => value.toString + ".rows"
```

Comment:
How about we leave this suffix, and just replace the return type with `LONG_TYPE_INFO`? This is part of the API as far as I understand. If we remove that some of the user programs might fail.

It would also be inline with the scala dsl. You didn't remove the `org.apache.flink.table.api.scala.ImplicitExpressionOperations#rows`

Reply:
The suffix expression `.rows` is still valid for expressions. But since the type does not exist anymore, this string can not be generated anymore.

Reply:
Sorry, totally my bad. I mistaken the class.